code/modules/fish/fish_items.dm

```diff
@@ -0,0 +1,218 @@
+	icon_state = "fish_food"
+	throwforce = 1
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+
+/obj/item/weapon/fishtools/fish_tank_brush
+	name = "aquarium brush"
+	desc = "A brush for cleaning the inside of aquariums. Contains a built-in odor neutralizer."
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "brush"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+	attack_verb = list("scrubbed", "brushed", "scraped")
+
+	suicide_act(mob/user)
+		to_chat(viewers(user), "<span class='warning'>[user] is vigorously scrubbing \himself raw with the [src.name]! It looks like \he's trying to commit suicide.</span>")
+		return(BRUTELOSS|FIRELOSS)
+
+//////////////////////////////////////////////
+//				Fish Items					//
+//////////////////////////////////////////////
+
+/obj/item/weapon/reagent_containers/food/snacks/shrimp
+	name = "shrimp"
+	desc = "A single raw shrimp."
+	icon = 'icons/obj/seafood.dmi'
+	icon_state = "shrimp_raw"
+	filling_color = "#FF1C1C"
+
+	New()
+		..()
+		desc = pick("Anyway, like I was sayin', shrimp is the fruit of the sea.", "You can barbecue it, boil it, broil it, bake it, saute it.")
+		reagents.add_reagent("NUTRIMENT", 1)
+		src.bitesize = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/feederfish
+	name = "feeder fish"
+	desc = "A tiny feeder fish. Sure doesn't look very filling..."
+	icon = 'icons/obj/seafood.dmi'
+	icon_state = "feederfish"
+	filling_color = "#FF1C1C"
+
+	New()
+		..()
+		reagents.add_reagent("NUTRIMENT", 1)
+		src.bitesize = 1
+
+/obj/item/weapon/fish
+	name = "fish"
+	desc = "A generic fish"
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "fish"
+	throwforce = 1
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+	force = 1
+	attack_verb = list("slapped", "humiliated", "hit", "rubbed")
+	hitsound = 'sound/effects/snap.ogg'
+
+/obj/item/weapon/fish/glofish
+	name = "glofish"
+	desc = "A small bio-luminescent fish. Not very bright, but at least it's pretty!"
+	icon_state = "glofish"
+
+/obj/item/weapon/fish/glofish/New()
+		..()
+		set_light(2,1,"#99FF66")
+
+/obj/item/weapon/fish/electric_eel
+	name = "electric eel"
+	desc = "An eel capable of producing a mild electric shock. Luckily it's rather weak out of water."
+	icon_state = "electric_eel"
+
+/obj/item/weapon/fish/shark
+	name = "shark"
+	desc = "Warning: Keep away from tornadoes."
+	icon_state = "shark"
+	hitsound = 'sound/weapons/bite.ogg'
+	force = 3
+
+/obj/item/weapon/fish/shark/attackby(var/obj/item/O, var/mob/user as mob)
+	if(istype(O, /obj/item/weapon/wirecutters))
+		to_chat(user, "You rip out the teeth of \the [src.name]!")
+		new /obj/item/weapon/fish/toothless_shark(get_turf(src))
+		new /obj/item/weapon/shard/shark_teeth(get_turf(src))
+		qdel(src)
+		return
+	..()
+
+/obj/item/weapon/fish/toothless_shark
+	name = "toothless shark"
+	desc = "Looks like someone ripped it's teeth out!"
+	icon_state = "shark"
+	hitsound = 'sound/effects/snap.ogg'
+
+/obj/item/weapon/shard/shark_teeth
```

Comment:
Also a bunch of other weird behaviors like being able to use shark teeth in ghetto weapons and spears because child of shard